lib/manual_and_sections_redirecter.rb

```diff
@@ -0,0 +1,67 @@
+class ManualAndSectionsRedirecter
+  def initialize(args)
+    @publishing_api = SpecialistPublisher.services(:publishing_api)
+    @logger = args[:logger] || STDOUT
+    @base_path = args[:base_path]
+    @destination = args[:destination]
+  end
+
+  def redirect
+    run
+  end
+
+private
+
+  attr_reader :base_path, :destination, :logger, :publishing_api
+
+  def run
+    manuals_response = publishing_api.get_content_items(publishing_app: "specialist-publisher", document_type: "manual", fields: %w(base_path content_id))
+
+    raise "Could not retrieve specialist-publisher manuals from publishing api" unless manuals_response.code == 200
```

Comment:
In these error cases, the response from the publishing api is being discarded. This could contain useful information that you wish to include in the output. It might be worth adding a method that will provide richer feedback in the error message if the status is non-200.